src/templates/post.tsx

```diff
@@ -299,6 +312,7 @@ const PageTemplate: React.SFC<PageTemplateProps> = props => {
                 <AuthorCard author={post.frontmatter.author} />
                 <PostFullFooterRight authorId={post.frontmatter.author.id} />
               </PostFullFooter>
+              <DiscussionEmbed config={disqusConfig} shortname="koodibar"/>
```

Comment:
@hungneox: move `shortname` as `koodibar` to ENV?

Reply:
Sounds good, please move to ENV